shade_ms/main.py

```diff
@@ -164,22 +109,37 @@ def get_conformal_list(name, force_type=None, default=None):
             sys.exit(1)
 
     # check chan slice
-    def parse_slice_spec(spec, name):
-        if spec:
+    try:
+        chanslice, chanslice_spec = parse_slice_spec(options.chan)
+    except ValueError:
+        parser.error(f"invalid selection --{'chan'} {options.chan}")
```

Comment:
f"--{'chan'}" seems to be a rather roundabout way of saying "--chan", what's the point?